messageapi/src/main/java/fi/hel/verkkokauppa/message/service/MessageService.java

```diff
@@ -29,6 +35,18 @@ public class MessageService {
     @Autowired
     private JavaMailSender javaMailSender;
 
+    @Autowired
+    private RestServiceClient restServiceClient;
+
+    @Value("${order.service.url:http://order-api:8080}")
+    private String orderServiceUrl;
+
+    @Value("${payment.service.url:http://payment-api:8080}")
+    private String paymentServiceUrl;
+
+    @Value("${merchant.service.url:http://merchant-api:8080}")
+    private String merchantServiceUrl;
```

Comment:
use common/src/main/java/fi/hel/verkkokauppa/common/configuration/ServiceUrls.java 

That have these same urls in one place no need to duplicate

Reply:
Are these even needed anymore in message api?